web/Controllers/LandlordController.cs

```diff
@@ -141,4 +140,44 @@ public IActionResult ViewProperties()
         var listOfProperties = databaseResult.Select(PropertyViewModel.FromDbModel).ToList();
         return View("Properties", new PropertiesDashboardViewModel(listOfProperties));
     }
+
+    [HttpGet]
+    [Route("/invite/link={inviteLink}")]
+    public ActionResult Invite(string inviteLink)

```

Comment:
This will work, but it's an unusual format for a URL. I think do one of

Have it in a query string, `/invite?link=123abc`

```cs
    [Route("/invite")]
    public ActionResult Invite([FromQuery] string inviteLink)
```

or have it as part of the path with no `=`, so `/invite/123abc`

```cs
    [Route("/invite/{inviteLink}")]
    public ActionResult Invite([FromRoute] string inviteLink)
```

Reply:
yeah ok I did the latter one.